src/scripts.js

```diff
@@ -1 +1,26 @@
-console.log("Hello World");
\ No newline at end of file
+let userRepo = new UserRepository(userData);
+
+const userName = document.querySelector('.user-name');
+const userAddress = document.querySelector('.user-address');
+const userEmail = document.querySelector('.user-email');
+const userStride = document.querySelector('.user-stride-length');
+const userStepGoal = document.querySelector('.user-step-goal');
+const userGoalAverage = document.querySelector('.step-goal-average');
+const friendsList = document.querySelector('.friends-list');
+const stepComparison = document.querySelector('.step-goal-comparison')
+
+function populateUserInfo(id) {
+  let user = new User(userRepo.getUserData(id))
+  userName.innerHTML = user.name;
+  userAddress.innerHTML = user.address;
+  userEmail.innerHTML = user.email;
+  userStride.innerHTML = user.strideLength;
+  userStepGoal.innerHTML = user.dailyStepGoal;
+  userGoalAverage.innerHTML = userRepo.getStepGoalAverage();
+  stepComparison.innerHTML = `Your goal is ${Math.floor((user.dailyStepGoal / userRepo.getStepGoalAverage()) * 100 - 100)}%
+    under/over the average step goal of all users`
```

Comment:
I think we can implement some conditional logic here to display the word 'over' or 'under' depending on the value instead of over/under:

```
let stepDifference = userRepo.getStepGoalAverage() - user.dailyStepGoal;
stepComparison.innerHTML = `Your goal is ${Math.abs(diff)} steps ${diff > 0 ? 'lower': 'higher'} than the average goal`;
```

also I think percentages might get weird, so showing them the number of steps difference is probably more useful

Reply:
Might need to add another condition for if it is the exact same as the average step goal, though I'm not sure the likelihood of that happening with a large dataset

Reply:
Fixed with a ternary